irr/src/CSkinnedMesh.cpp

```diff
@@ -596,6 +597,14 @@ IMeshBuffer *CSkinnedMesh::getMeshBuffer(const video::SMaterial &material) const
 	return 0;
 }
 
+u32 CSkinnedMesh::getTextureSlot(u32 meshbufNr) const {
```

Comment:
```suggestion
u32 CSkinnedMesh::getTextureSlot(u32 meshbufNr) const
{
```

And the same for all other occurrences.